data/com.redhat.Yggdrasil1.Worker1.foreman.service.in

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=rhc-worker-playbook worker service
```

Comment:
I don't see a short description for this repo on GH, but the first line in README reads:
> This is a worker service for yggdrasil that can act as pull client for Foreman Remote Execution.

So something like this perhaps?
```suggestion
Description=An yggdrasil worker service to act as a pull client for Foreman Remote Execution
```

@adamruzicka is that a good summary we should also use for the GH repo?

Reply:
Oops. This was a copy-paste error.